bin/get_fulltext.py

```diff
@@ -2,6 +2,23 @@
 
 """get_fulltext CLI command"""
```

Comment:
Is there a reason, why you do not put the copyright notice below the docstring for the module?

Reply:
discussed offline that we can keep a single line docstring above the copyright, though each way around has its pros and cons